lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/services/FeatureService.java

```diff
@@ -72,8 +73,11 @@ public static FeatureService getInstance() {
     private List<Feature> defaultFeatureList;
     private long featureUpdateTime;
 
+    private FeatureListGraph featureListGraph;
+
     private FeatureService() {
         featureCache = new HashMap<>();
+        featureListGraph = new FeatureListGraph();
```

Comment:
Does this need to be a Map just like `featureCache`? This graph should be specific to the Liberty installation. If they have multiple projects with different runtimes installed, need to use the correct `FeatureListGraph`. Even caching by `ol-23.0.0.6` may not be sufficient in the long run, if they have two different projects using that runtime with a different set of features installed. Maybe the graph should be stored in the `LibertyWorkspace`?